cli/tests/testdata/npm/dynamic_import/other.ts

```diff
@@ -2,3 +2,10 @@ console.log("B");
 const chalk = (await import("npm:chalk@5")).default;
 
 console.log(chalk.green("C"));
+
+try {
+  // Trying to import a devDependency should result in an error
+  await import("xo");
+} catch (e) {
+  console.error("devDependency import failed:", e);
```

Comment:
This could use better error message instead of `Relative import path "xo" is not prefix with /, ./ or ../`